tests/integration/steps/integration_steps.go

```diff
@@ -133,43 +132,70 @@ func DeployClusterWithValues(t *testing.T, namespace, options, customValues stri
 			"medusa.storage_properties.host": fmt.Sprintf("%s.minio.svc.cluster.local", serviceName),
 		}
 	}
+	helmValues["cassandra.datacenters[0].size"] = strconv.Itoa(nodes)
+	helmValues["cassandra.datacenters[0].name"] = datacenterName
 	deployCluster(t, namespace, customValues, helmValues)
 }
 
-func DeployClusterWithValuesAndHeapSettings(t *testing.T, namespace, options, cassandraHeap, stargateHeap, customValues string) {
-	log.Printf("I can deploy a cluster with %s options, %s Cassandra Heap and %s Stargate heap using the %s values", options, cassandraHeap, stargateHeap, customValues)
-
-	splitOptions := strings.Split(options, "-")
-	medusaEnabled := "true"
-	reaperEnabled := "true"
-	monitoringEnabled := "true"
+func ScaleClusterWithValues(t *testing.T, namespace, options, customValues string, nodes int) {
+	log.Printf("Deploying a cluster with %s options using the %s values", options, customValues)
 
-	if Find(splitOptions, "nomedusa") {
-		medusaEnabled = "false"
+	helmValues := map[string]string{}
+	if options == "default" {
+		helmValues = map[string]string{
+			"reaper.ingress.host": "repair.127.0.0.1.nip.io",
+		}
 	}
-
-	if Find(splitOptions, "noreaper") {
-		reaperEnabled = "false"
+	if options == "minio" {
+		serviceName := MinioServiceName(t)
+		helmValues = map[string]string{
+			"medusa.storage_properties.host": fmt.Sprintf("%s.minio.svc.cluster.local", serviceName),
+		}
 	}
+	helmValues["cassandra.datacenters[0].size"] = strconv.Itoa(nodes)
+	helmValues["cassandra.datacenters[0].name"] = datacenterName
+	scaleCluster(t, namespace, customValues, helmValues)
+}
 
-	if Find(splitOptions, "nomonitoring") {
-		monitoringEnabled = "false"
+func scaleCluster(t *testing.T, namespace, customValues string, helmValues map[string]string) {
```

Comment:
I find the name `scaleCluster` misleading. I say that because I looked over the function and it looks like it is doing a `helm upgrade` and then blocking until the CassandraDatacenter is ready. If my observation is correct, could we rename it to make it more clear?

Reply:
What would you name it instead?

Reply:
It's removed now and fully integrated into the existing `DeployCluster***` functions.